site/docs/status.md

```diff
@@ -0,0 +1,358 @@
+|----------------|------|-----------|------|----|
+| createView     | Y    | N         | N    | N  |
+| dropView       | Y    | N         | N    | N  |
+| listView       | Y    | N         | N    | N  |
+| viewExists     | Y    | N         | N    | N  |
+| replaceView    | Y    | N         | N    | N  |
+| renameView     | Y    | N         | N    | N  |
+
+#### Namespace Operations
+
+| Namespace Operation       | Java | PyIceberg | Rust | Go |
+|---------------------------|------|-----------|------|----|
+| listNamespaces            | Y    | N         | N    | N  |
+| createNamespace           | Y    | N         | N    | N  |
+| dropNamespace             | Y    | Y         | Y    | Y  |
+| namespaceExists           | Y    | N         | N    | N  |
+| renameNamespace           | Y    | N         | N    | N  |
+| updateNamespace           | Y    | N         | N    | N  |
+| updateNamespaceProperties | Y    | Y         | Y    | Y  |
+| loadNamespaceMetadata     | Y    | N         | N    | N  |
+
+### Glue Catalog
+
+#### Table Spec V1
+
+| Table Operation | Java | PyIceberg | Rust | Go |
+|-----------------|------|-----------|------|----|
+| listTable       | Y    | Y         | Y    | Y  |
+| createTable     | Y    | Y         | Y    | Y  |
+| dropTable       | Y    | Y         | Y    | Y  |
+| loadTable       | Y    | Y         | Y    | Y  |
+| updateTable     | Y    | Y         | Y    | Y  |
+| renameTable     | Y    | Y         | Y    | Y  |
+| tableExists     | Y    | Y         | Y    | Y  |
+
+#### Table Spec V2
+
+| Table Operation | Java | PyIceberg | Rust | Go |
+|-----------------|------|-----------|------|----|
+| listTable       | Y    | Y         | Y    | Y  |
+| createTable     | Y    | Y         | Y    | Y  |
+| dropTable       | Y    | Y         | Y    | Y  |
+| loadTable       | Y    | Y         | Y    | Y  |
+| updateTable     | Y    | Y         | Y    | Y  |
+| renameTable     | Y    | Y         | Y    | Y  |
+| tableExists     | Y    | Y         | Y    | Y  |
+
+#### View Spec V1
+
+| View Operation | Java | PyIceberg | Rust | Go |
+|----------------|------|-----------|------|----|
+| createView     | Y    | N         | N    | N  |
+| dropView       | Y    | N         | N    | N  |
+| listView       | Y    | N         | N    | N  |
+| viewExists     | Y    | N         | N    | N  |
+| replaceView    | Y    | N         | N    | N  |
+| renameView     | Y    | N         | N    | N  |
+
+#### Namespace Operations
+
+| Namespace Operation       | Java | PyIceberg | Rust | Go |
+|---------------------------|------|-----------|------|----|
+| listNamespaces            | Y    | N         | N    | N  |
+| createNamespace           | Y    | N         | N    | N  |
+| dropNamespace             | Y    | N         | N    | N  |
+| namespaceExists           | Y    | N         | N    | N  |
+| renameNamespace           | Y    | N         | N    | N  |
+| updateNamespaceProperties | Y    | Y         | Y    | Y  |
```

Comment:
GlueCatalog doesn't seem to have 1renameNamespace`

```suggestion
| namespaceExists           | Y    | N         | N    | N  |
| updateNamespaceProperties | Y    | Y         | Y    | Y  |
```

https://github.com/apache/iceberg/blob/main/aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java